Cargo.toml

```diff
@@ -32,6 +32,7 @@ prettytable-rs = { version = "0.10.0"}
 urlencoding = { version = "2.1.3" }
 actix-cors = { version = "0.6.5" }
 url = { version = "2.5.0", features = ["serde"] }
+html_parser = { version = "0.7.0" }
```

Comment:
I know very little about Rust HTML5 parsers. Is this the right one for the project vs. `html5ever`, etc.? (Really, no informed opinion here, just curiosity.)

Reply:
This is just my go-to parser, but after some research looks like using an `html5ever`-based parser is going to be better.
I'll use `scraper`, an `html5ever`-based crate with `selectors` :)

Reply:
Wow! This ended up so clean! :)